.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -34,7 +34,7 @@ body:
     id: reproduce
     attributes:
       label: Steps to Reproduce
-      description: Tell us how to reproduce this behaviour. Ideally, this should take the form of a [Minimum Workable Example](https://stackoverflow.com/help/minimal-reproducible-example)
+      description: Tell us how to reproduce this behaviour. Ideally, this should take the form of a minimum workable example.
```

Comment:
We should tell users what a minimum workable example is

Reply:
Done